Hyperobjekt/bil-map
# Add ability to export all experiment site data as CSV

_Note: This PR ladders off #40. Thus, it quietly includes all the commits from that PR, so we should plan to review/merge #40 before this one. Once we do, this PR will automatically change its base to `master`._

This PR adds the ability to export all experiment sites via CSV. Accessing the export still requires selecting an experiment site, but now the "Export" button (which is renamed "Export Selected" for clarity) has an "Export All" below it:

<img width="253" alt="Screenshot of our Export interface showing a new Export All button" src="https://user-images.githubusercontent.com/4731/205183861-6cdf034b-00ba-4865-a167-62d4b18b24ea.png">

**Testing:**
1. Select a site to trigger the selection interface
2. Scroll down to the Export/Share section
3. Click "Export Selected" and ensure the download CSV is named `SelectedBILData.csv`) contains only the selection data
4. Click "Export All" and ensure the downloaded CSV is named `BILData.csv` and contains all experiment site data

Closes #41
